app/components/questions/RideTripsOhare.js

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import { FailureNotification } from '../../components/Notification'
+import {
+  Formatter,
+  calculatePercentChange,
+} from '../../site/metrics'
+
+const RIDETRIPS_ENDPOINT = `${process.env.NEXT_PUBLIC_API}/question/ridetrips`;
+  
+  const MAX_PCT_CHANGE = -0.76;
+  const RIDETRIPS_DROPOFF_ALL = [
+  {
+    key: "dropoff_community_area",
+    name: "Area Number",
+  },
+  {
+    key: "area_name",
+    name: "Dropoff Location",
+  },
+  {
+    key: "total_trips_2019",
+    name: "Dropoffs in 2019",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "total_trips_2020",
+    name: "Dropoffs in 2020",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "pct_change",
+    name: "Change",
+    format: Formatter.percentChangeWithNoDecimal,
+    rowClasses: ["right"],
+    rgb: "209, 31, 0",
+    alpha: (v) => (v / MAX_PCT_CHANGE),
+  },
+  ];
+
+function addTotalTripsYear(before,since) {
+
+  //retrieving key:value pairs
+  var dropoffs = before.map(({
+    total_trips: total_trips_2019,
+    dropoff_community_area: dropoff_community_area,
+    area_name: area_name
+  }) => ({
+    total_trips_2019,
+    dropoff_community_area,
+    area_name: area_name
+  }));
+  
+  const dropoffs_2020 = since.map(({
+    total_trips: total_trips_2020,
+
+  }) => ({
+    total_trips_2020,
+
+  }));
+  
+  //adding 2020 trips within the same object as the 2019 trips
```

Comment:
This also assumes that the results are in the same order by area number. Would you rather write this logic on the backend in Python or on the frontend in JavaScript?